lib/route.coffee

```diff
@@ -159,11 +159,12 @@ class @Route
     auth = @api.config.auth.user.call(endpointContext)
 
     # Get the user from the database
-    if not auth?.user and auth?.userId and auth?.token
-      userSelector = {}
-      userSelector._id = auth.userId
-      userSelector[@api.config.auth.token] = auth.token
-      auth.user = Meteor.users.findOne userSelector
+    if not auth?.user
+      if auth?.userId and auth?.token
```

Comment:
The original code was a little hard to parse as a human. In general, I prefer to explicitly define the logic in order to ensure all humans understand it.
In other words, a human could read the original code as:

    not ( user and userId and token )

When the CPU would read it as (I believe):

    ( not user ) and userId and token

This change is purely to make it easier for a human to understand the desired logic without question.